src/appMain/sdl_preloaded_pt.json

```diff
@@ -587,7 +587,8 @@
                             "odometer",
                             "tirePressure",
                             "vin",
-                            "wiperStatus"
+                            "wiperStatus",
+                            "handsOffSteering"
```

Comment:
@LitvinenkoIra to move it to the "DrivingCharacteristics-3" group

Reply:
Done in the dd77569bc33db24ce371f3388bf730a5a197e684